content/intro-to-storybook/vue/pt/get-started.md

```diff
@@ -64,15 +62,14 @@ O CSS compilado encontra-se disponível [aqui](https://github.com/chromaui/learn
 ![Interface Utilizador Taskbox](/intro-to-storybook/ss-browserchrome-taskbox-learnstorybook.png)
 
 <div class="aside">
-  Se for necessária alguma alteração aos elementos de estilo, os ficheiros LESS originais encontram-se disponíveis no repositório Github.
+  Se você quiser alterar o estilo, os arquivos LESS originais encontram-se disponíveis no repositório GitHub.
```

Comment:
A better wording for this could be something like:
" Se quiser alterar os estilos, os arquivos ou ficheiros LESS originais encontram-se disponíveis no repositório GitHub."

I'm sugesting a pluralization due to the fact that you can not only change the look of the app, but also icons, fonts and so on. And also including both variations of the word to avoid migration to one language.

Reply:
Makes more sense this way.